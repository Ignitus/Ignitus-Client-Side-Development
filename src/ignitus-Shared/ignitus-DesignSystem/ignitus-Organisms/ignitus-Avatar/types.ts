export type avatar = {
  name: string;
  src: string;
};

/** As AvatarImage represents img feel free to add more props here, as <img /> supports lot of other attributes. */
export type AvatarImageProps = {
  src: string;
  alt: string;
};
