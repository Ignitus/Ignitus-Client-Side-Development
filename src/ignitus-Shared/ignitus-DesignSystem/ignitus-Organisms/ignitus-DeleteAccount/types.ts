export interface PropTypes {
  username: string;
}
