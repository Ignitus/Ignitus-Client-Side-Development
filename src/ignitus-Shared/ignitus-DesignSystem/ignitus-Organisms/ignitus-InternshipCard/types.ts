export type CardTypes = {
  ImageLink: string;
  Title: string;
  Position?: string;
  Place?: string;
  Company?: string;
  Description?: string;
  type?: string;
  JobType?: string;
};

export type CardProps = {
  ImageLink: string;
  Title: string;
  Position?: string;
  Place?: string;
  Company?: string;
  Description?: string;
  type: string;
  JobType?: string;
};

export type StyleProps = {
  size?: string;
};
