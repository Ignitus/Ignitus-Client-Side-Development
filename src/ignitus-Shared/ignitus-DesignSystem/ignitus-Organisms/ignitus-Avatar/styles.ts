import styled from '@emotion/styled';

export const AvatarImage = styled.img`
  max-width: 9rem;
`;
