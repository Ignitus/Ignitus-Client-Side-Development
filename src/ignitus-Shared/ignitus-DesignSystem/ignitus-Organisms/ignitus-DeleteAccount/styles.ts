import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import { Paragraph } from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';
import { Icon } from '../../../ignitus-Utilities/Components/icon';
import { DefaultCheckbox } from '../../..';
import { RoundedButton } from '../../ignitus-Atoms/buttons';

export const DeleteContainer = styled.div`
  background-color: ${C.White};
  background-size: contain;
  border-radius: 1rem;
  padding: 2rem;
  display: inline-flex;
  flex-direction: column;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  width: 40rem;
`;

export const TopSection = styled(flexibleRowDiv)`
  justify-content: space-between;
`;

export const MiddleSection = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
`;

export const BottomSection = styled(flexibleRowDiv)`
  justify-content: flex-end;
  margin-top: 2rem;
`;

export const StyledIcon = styled(Icon)`
  width: 1.4rem;
  fill: ${props => props.color};
  transition: 0.3s;
  flex: 1;
  margin-top: 0.2rem;
`;

export const IconContainer = styled.button`
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: ${props => props.color};
  outline-color: transparent;
`;

export const StyledParagraph = styled(Paragraph)`
  padding: 1rem 0;
`;

export const StyledCheckbox = styled(DefaultCheckbox)`
  font-size: ${F.SM};
  margin-bottom: 1rem;
`;

export const StyledTextArea = styled.textarea`
  background-color: ${C.GreyLight};
  padding: 1rem;
  color: ${C.GreySecondaryText};
  font-weight: ${F.SemiBold};
  font-size: ${F.MD};
  border: none;
  border-radius: 1rem;

  ::placeholder {
    color: ${C.GreySecondaryText};
    font-weight: ${F.SemiBold};
    font-size: ${F.MD};
    opacity: 0.5;
  }
`;

export const StyledRoundedButton = styled(RoundedButton)`
  font-weight: ${F.Bold};
  box-shadow: 0 6px 12px ${C.boxShadowColorLight};
`;
