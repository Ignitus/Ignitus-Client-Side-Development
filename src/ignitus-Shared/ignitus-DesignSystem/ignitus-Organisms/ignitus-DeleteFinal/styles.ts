import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import { Paragraph } from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';
import { Icon } from '../../../ignitus-Utilities/Components/icon';
import { RoundedButton } from '../../ignitus-Atoms/buttons';

export const DeleteContainer = styled.div`
  background-color: ${C.White};
  background-size: contain;
  border-radius: 1rem;
  display: inline-flex;
  flex-direction: column;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  width: 45rem;
  padding: 2rem;
`;

export const TopSection = styled(flexibleRowDiv)`
  justify-content: space-between;
`;

export const MiddleSection = styled.div`
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
`;

export const BottomSection = styled(flexibleRowDiv)`
  margin-top: 1rem;
  justify-content: flex-end;
`;

export const StyledIcon = styled(Icon)`
  width: 1.4rem;
  fill: ${props => props.color};
  transition: 0.3s;
  flex: 1;
  margin-top: 0.2rem;
`;

export const IconContainer = styled.button`
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: ${props => props.color};
  outline-color: transparent;
`;

export const StyledParagraph = styled(Paragraph)`
  padding: 0.5rem 0;
`;

export const StyledText = styled(Paragraph)`
  font-weight: ${F.Bold};
  padding: 0.5rem 0;
`;

export const StyledRoundedButton = styled(RoundedButton)`
  box-shadow: 0 6px 12px ${C.boxShadowColorLight};
`;
