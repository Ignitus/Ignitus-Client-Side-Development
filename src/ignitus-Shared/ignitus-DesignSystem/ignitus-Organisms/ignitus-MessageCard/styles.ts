import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import * as T from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv } from '../../shared';

export const MessageContainer = styled(flexibleRowDiv)`
  border-radius: 1rem;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  background-color: ${C.White};
  box-sizing: border-box;
  padding: 1rem;
  width: 100%;
  margin-bottom: 1rem;
`;

export const ImageContainer = styled.div`
  position: relative;
  padding: 0.5rem;

  img {
    border-radius: 50%;
    width: 4rem;
    max-width: 100%;
    max-height: 100%;
  }

  svg {
    position: absolute;
    bottom: 0;
    right: 0;
    stroke: ${C.White};
    fill: #219653;
    height: 2rem;
    width: 2rem;
  }
`;

export const MessageSubTitle = styled(T.Paragraph)`
  margin-left: 1rem;
`;

export const Sender = styled(T.Paragraph)`
  font-weight: ${F.Bold};
  font-size: ${F.MD};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const SenderPosition = styled(T.Paragraph)`
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const MessageDateTitle = styled.div`
  text-align: right;
  margin-left: auto;
`;

export const Details = styled(T.Paragraph)`
  font-family: Open Sans;
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.GreyOne};
  margin-bottom: 0.5rem;
`;
