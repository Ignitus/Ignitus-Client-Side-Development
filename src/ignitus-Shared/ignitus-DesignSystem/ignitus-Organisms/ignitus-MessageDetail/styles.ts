import styled from '@emotion/styled';
import * as C from '../../ignitus-Atoms/colors';
import * as T from '../../ignitus-Atoms/typography';
import * as F from '../../ignitus-Atoms/fonts';
import { flexibleRowDiv, flexibleColDiv, Paragraph } from '../../shared';
import { DefaultButtonWithIcon } from '../../..';

export const MessageContainer = styled(flexibleColDiv)`
  border-radius: 1rem;
  box-shadow: 0 2px 4px 0 ${C.boxShadowColor};
  background-color: ${C.White};
  box-sizing: border-box;
  width: 100%;
  margin-bottom: 1rem;
`;

export const ImageContainer = styled.div`
  position: relative;
  padding: 0.5rem;

  img {
    border-radius: 50%;
    width: 4rem;
    max-width: 100%;
    max-height: 100%;
  }

  svg {
    position: absolute;
    bottom: 0;
    right: 0;
    stroke: ${C.White};
    fill: #219653;
    height: 2rem;
    width: 2rem;
  }
`;

export const MessageSubTitle = styled(T.Paragraph)`
  margin-left: 1rem;
`;

export const Sender = styled(T.Paragraph)`
  font-weight: ${F.Bold};
  font-size: ${F.MD};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const SenderPosition = styled(T.Paragraph)`
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.IgnitusBlue};
  margin-bottom: 0.5rem;
`;

export const MessageDateTitle = styled.div`
  text-align: right;
  margin-left: auto;
`;

export const Details = styled(T.Paragraph)`
  font-family: Open Sans;
  font-weight: ${F.Normal};
  font-size: ${F.SM};
  color: ${C.GreyOne};
  margin-bottom: 0.5rem;
`;

export const Message = styled(Paragraph)``;

export const TopSection = styled(flexibleRowDiv)`
  box-sizing: border-box;
  padding: 1rem 1rem 0 1rem;
  width: 100%;
`;

export const MiddleSection = styled(flexibleRowDiv)`
  box-sizing: border-box;
  padding: 1rem;
  width: 100%;
`;

export const BottomSection = styled(flexibleRowDiv)`
  box-sizing: border-box;
  padding-right: 1rem;
  width: 100%;
  margin-bottom: 1rem;
  justify-content: flex-end;
`;

export const StyledDefaultButtonWithIcon = styled(DefaultButtonWithIcon)`
  flex-direction: row-reverse;
  box-shadow: 0 6px 12px ${C.boxShadowColorLight};
  border-radius: 1rem;
  font-weight: ${F.Bold};
  svg {
    margin-right: 0.2rem;
  }
`;

export const StyledLine = styled.hr`
  width: -webkit-fill-available;
`;
