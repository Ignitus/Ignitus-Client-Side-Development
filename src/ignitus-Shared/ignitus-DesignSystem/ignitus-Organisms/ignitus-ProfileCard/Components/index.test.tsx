import React from 'react';
import { mount, render } from 'enzyme';
import emotionSerializer, { matchers } from 'jest-emotion';
import { UserProfileCard, IconContent, DropDownButton } from './index';
import * as S from '../styles';
import { Props } from '../types';
import { AppIcon } from '../../../../types/iconsTypes/iconEnums';

const props: Props = {
  content: 'Send Message',
  name: AppIcon.QuestionAnswerIcon,
};

expect.addSnapshotSerializer(emotionSerializer);
expect.extend(matchers);

it('should rendered', () => {
  const tree = render(<UserProfileCard />);
  expect(tree).toMatchSnapshot();
});

describe('<UserProfileCard />', () => {
  let wrapper;
  it('should render Icon, Paragraph, Avatar', () => {
    wrapper = mount(<UserProfileCard />);

    expect(wrapper.find(S.Icon).length).toBe(8);
    expect(wrapper.find(IconContent).length).toBe(3);
    expect(wrapper.find(S.Paragraph).length).toBe(4);
    expect(wrapper.find(S.Avatar).length).toBe(1);
  });

  it('should render IconContent correctly', () => {
    wrapper = mount(<IconContent {...props} />);

    expect(wrapper.find(S.Paragraph).text()).toBe(props.content);
    expect(wrapper.find(S.Icon).prop('name')).toBe(props.name);
  });

  it('should render DropDownButton correctly', () => {
    wrapper = mount(<DropDownButton />);

    expect(wrapper.find(S.DropDownMenuListItem).length).toBe(3);
    expect(wrapper.find(S.DropDownBtn).length).toBe(1);
  });
});
