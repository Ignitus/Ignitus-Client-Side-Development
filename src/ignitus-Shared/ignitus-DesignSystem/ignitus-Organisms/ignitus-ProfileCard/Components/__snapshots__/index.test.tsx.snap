// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rendered 1`] = `
.emotion-35 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 60rem;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  padding: 1rem 2rem;
  border-radius: 1rem;
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.05);
  background-color: #ffffff;
  color: #000066;
}

.emotion-2 {
  text-align: end;
}

.emotion-0 {
  font-size: 16px;
  font-weight: 300;
  color: #4D4D4D;
  font-weight: 400;
  display: inline;
  margin: 0 0.5rem 0 0;
}

.emotion-1 {
  width: 1.5rem;
  fill: #000066;
}

.emotion-34 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  padding: 1rem;
}

.emotion-16 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-flex: 2;
  -ms-flex: 2;
  flex: 2;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.emotion-3 {
  width: 10rem;
  border-radius: 1rem;
}

.emotion-15 {
  margin-left: 4rem;
}

.emotion-4 {
  font-weight: 500;
  font-size: 32px;
  color: #000066;
  font-size: 32px;
  font-weight: 600;
}

.emotion-5 {
  font-weight: 400;
  font-size: 20px;
  color: #000066;
  margin-bottom: 1rem;
  margin-top: 1rem;
}

.emotion-8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0.6rem 0;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.emotion-7 {
  font-size: 16px;
  font-weight: 300;
  color: #000066;
  font-weight: 400;
  display: inline;
  margin: 0 0 0 0.5rem;
}

.emotion-14 {
  position: relative;
  margin-top: 1rem;
}

.emotion-13 {
  font-weight: bold;
  color: #9C9C9C;
  cursor: pointer;
  display: inline-block;
  padding-bottom: 10px;
  -webkit-transition: all 0.4s;
  transition: all 0.4s;
  width: -webkit-fit-content;
  width: -moz-fit-content;
  width: fit-content;
}

.emotion-13:hover ul {
  display: block;
}

.emotion-12 {
  z-index: 3;
  position: absolute;
  top: 30px;
  left: 0;
  border-radius: 10px 0 10px 10px;
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.05);
  background-color: #ffffff;
  color: #000066;
  display: none;
  font-weight: 400;
}

.emotion-12:hover {
  display: block;
}

.emotion-9 {
  padding: 0.5em 1em;
  font-size: 14px;
  width: 200px;
}

.emotion-9:first-of-type {
  border-top-left-radius: 10px;
}

.emotion-9:last-child {
  border-bottom-left-radius: 10px;
  border-bottom-right-radius: 10px;
}

.emotion-9:hover {
  color: #ffffff;
  background-color: #000066;
  cursor: pointer;
}

.emotion-33 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.emotion-25 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-bottom: 2rem;
}

.emotion-18 {
  margin-right: 2rem;
  display: inline-block;
}

.emotion-32 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 100%;
}

<div
  class="emotion-35"
>
  <div
    class="emotion-2"
  >
    <p
      class="emotion-0"
    >
      Track Sophi progress
    </p>
    <svg
      aria-label="Bookmark Icon"
      class="emotion-1"
      viewBox="0 0 384 512"
      xmlns="http://www.w3.org/2000/svg"
    >
      <title>
        Bookmark Icon
      </title>
      <path
        d="M336 0H48C21.49 0 0 21.49 0 48v464l192-112 192 112V48c0-26.51-21.49-48-48-48zm0 428.43l-144-84-144 84V54a6 6 0 0 1 6-6h276c3.314 0 6 2.683 6 5.996V428.43z"
      />
    </svg>
  </div>
  <div
    class="emotion-34"
  >
    <div
      class="emotion-16"
    >
      <div>
        <img
          alt="ProfileLogo"
          class="emotion-3"
          src="https://storage.googleapis.com/ignitus_assets/ig-avatars/eugene.png"
        />
      </div>
      <div
        class="emotion-15"
      >
        <h1
          class="emotion-4"
        >
          Sophia Carter
        </h1>
        <h4
          class="emotion-5"
        >
          Literature Student
        </h4>
        <div
          class="emotion-8"
        >
          <svg
            aria-label="Location Icon"
            class="emotion-1"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Location Icon
            </title>
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <path
              d="M12 2C8.13 2 5 5.13 5 9c0 4.17 4.42 9.92 6.24 12.11.4.48 1.13.48 1.53 0C14.58 18.92 19 13.17 19 9c0-3.87-3.13-7-7-7zm0 9.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z"
            />
          </svg>
          <p
            class="emotion-7"
          >
            Canada
          </p>
        </div>
        <div
          class="emotion-14"
        >
          <div
            class="emotion-13"
          >
            More ...
            <ul
              class="emotion-12"
            >
              <li
                class="emotion-9"
              >
                Ask For Recommendation
              </li>
              <li
                class="emotion-9"
              >
                Share Profile
              </li>
              <li
                class="emotion-9"
              >
                Block
              </li>
            </ul>
          </div>
        </div>
      </div>
    </div>
    <div
      class="emotion-33"
    >
      <div
        class="emotion-25"
      >
        <a
          class="emotion-18"
          href=""
        >
          <svg
            aria-label="Facebook Icon"
            class="emotion-1"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Facebook Icon
            </title>
            <path
              d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
            />
          </svg>
        </a>
        <a
          class="emotion-18"
          href=""
        >
          <svg
            aria-label="LinkedIn Icon"
            class="emotion-1"
            role="img"
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              LinkedIn Icon
            </title>
            <path
              d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
            />
          </svg>
        </a>
        <a
          class="emotion-18"
          href=""
        >
          <svg
            aria-hidden="true"
            class="emotion-1"
            data-icon="github"
            data-prefix="fab"
            focusable="false"
            role="img"
            viewBox="0 0 496 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Github Icon
            </title>
            <path
              d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
            />
          </svg>
        </a>
        <a
          class="emotion-18"
          href=""
        >
          <svg
            aria-label="Google Plus Icon"
            class="emotion-1"
            role="img"
            viewBox="0 0 640 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              Google Plus Icon
            </title>
            <path
              d="M386.061 228.496c1.834 9.692 3.143 19.384 3.143 31.956C389.204 370.205 315.599 448 204.8 448c-106.084 0-192-85.915-192-192s85.916-192 192-192c51.864 0 95.083 18.859 128.611 50.292l-52.126 50.03c-14.145-13.621-39.028-29.599-76.485-29.599-65.484 0-118.92 54.221-118.92 121.277 0 67.056 53.436 121.277 118.92 121.277 75.961 0 104.513-54.745 108.965-82.773H204.8v-66.009h181.261zm185.406 6.437V179.2h-56.001v55.733h-55.733v56.001h55.733v55.733h56.001v-55.733H627.2v-56.001h-55.733z"
            />
          </svg>
        </a>
      </div>
      <div
        class="emotion-32"
      >
        <div
          class="emotion-8"
        >
          <svg
            aria-label="QuestionAnswer Icon"
            class="emotion-1"
            role="img"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              QuestionAnswer Icon
            </title>
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <path
              d="M20 6h-1v8c0 .55-.45 1-1 1H6v1c0 1.1.9 2 2 2h10l4 4V8c0-1.1-.9-2-2-2zm-3 5V4c0-1.1-.9-2-2-2H4c-1.1 0-2 .9-2 2v13l4-4h9c1.1 0 2-.9 2-2z"
            />
          </svg>
          <p
            class="emotion-7"
          >
            Send Message
          </p>
        </div>
        <div
          class="emotion-8"
        >
          <svg
            aria-label="File Icon"
            class="emotion-1"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              File Icon
            </title>
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <path
              d="M14.59 2.59c-.38-.38-.89-.59-1.42-.59H6c-1.1 0-2 .9-2 2v16c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8.83c0-.53-.21-1.04-.59-1.41l-4.82-4.83zM15 18H9c-.55 0-1-.45-1-1s.45-1 1-1h6c.55 0 1 .45 1 1s-.45 1-1 1zm0-4H9c-.55 0-1-.45-1-1s.45-1 1-1h6c.55 0 1 .45 1 1s-.45 1-1 1zm-2-6V3.5L18.5 9H14c-.55 0-1-.45-1-1z"
            />
          </svg>
          <p
            class="emotion-7"
          >
            Resume
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
